src/main/java/core/basesyntax/MathMethods.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface MathMethods {
```

Comment:
`MathMethods` sounds like util class with a lot of static methods to make some math operations. Better call it `AreaCalculator` and introduce `calculateArea` method.